torchvision/prototype/transforms/functional/_geometry.py

```diff
@@ -815,18 +811,18 @@ def crop_bounding_box(
 ) -> Tuple[torch.Tensor, Tuple[int, int]]:
     # TODO: Investigate if it makes sense from a performance perspective to have an implementation for every
```

Comment:
Yes we don't have *every conversion*. I remember when adding I had discussion with Francisco that all conversions can simply go through intermediate xyxy conversion for sake of less complications.